core/lib/services/intersection.ts

```diff
@@ -0,0 +1,77 @@
+import type {ReadableSignal} from '@amadeus-it-group/tansu';
+import {asReadable, derived} from '@amadeus-it-group/tansu';
+import {writablesForProps, type ValuesOrStores} from './stores';
+import {noop} from '../utils';
+
+export type IntersectionPropsConfig<T extends object> = ReadableSignal<Partial<T>> | ValuesOrStores<T>;
```

Comment:
Please do not duplicate this definition that already exists as `PropsConfig` in `stores.ts`